help([==[

Description
===========
Bioconductor provides tools for the analysis and coprehension
 of high-throughput genomic data.


More information
================
 - Homepage: https://bioconductor.org


Included extensions
===================
admisc-0.5, affy-1.64.0, affycoretools-1.58.4, affyio-1.56.0,
AgiMicroRna-2.36.0, ALDEx2-1.18.0, ALL-1.28.0, annaffy-1.58.0,
annotate-1.64.0, AnnotationDbi-1.48.0, AnnotationFilter-1.10.0,
AnnotationForge-1.28.0, AnnotationHub-2.18.0, aroma.affymetrix-3.2.0,
aroma.apd-0.6.0, aroma.core-3.2.1, aroma.light-3.16.0, askpass-1.1,
ATACseqQC-1.10.2, ballgown-2.18.0, batchelor-1.2.4, baySeq-2.20.0,
beachmat-2.2.1, Biobase-2.46.0, BiocFileCache-1.10.2, BiocGenerics-0.32.0,
BiocManager-1.30.10, BiocNeighbors-1.4.1, BiocParallel-1.20.1,
BiocSingular-1.2.2, BiocVersion-3.10.1, biomaRt-2.42.0, biomformat-1.14.0,
Biostrings-2.54.0, biovizBase-1.34.1, BSgenome-1.54.0,
BSgenome.Hsapiens.UCSC.hg19-1.4.0, BSgenome.Hsapiens.UCSC.hg38-1.4.1,
BSgenome.Mmusculus.UCSC.mm10-1.4.0, bumphunter-1.28.0, CAMERA-1.42.0,
Category-2.52.1, CGHbase-1.46.0, ChIPpeakAnno-3.20.1, clue-0.3-57,
ComplexHeatmap-2.2.0, ConsensusClusterPlus-1.50.0, cummeRbund-2.28.0,
cytolib-1.8.0, CytoML-1.12.1, dada2-1.14.1, DeconRNASeq-1.28.0, DEGseq-1.40.0,
DelayedArray-0.12.2, DelayedMatrixStats-1.8.0, derfinder-1.20.0,
derfinderHelper-1.20.0, DESeq2-1.26.0, diffcyt-1.6.6, DNAcopy-1.60.0,
dqrng-0.2.1, DropletUtils-1.6.1, dupRadar-1.16.0, DynDoc-1.64.0,
EBImage-4.28.1, edgeR-3.28.0, ensembldb-2.10.2, fda-5.1.4,
FDb.InfiniumMethylation.hg19-2.2.0, fgsea-1.12.0, flowClust-3.24.0,
flowCore-1.52.1, FlowSOM-1.18.0, flowStats-3.44.0, flowViz-1.50.0,
flowWorkspace-3.34.1, fs-1.3.1, gcrma-2.58.0, gdsfmt-1.22.0,
genefilter-1.68.0, geneLenDataBase-1.22.0, geneplotter-1.64.0, GENESIS-2.16.1,
GENIE3-1.8.0, GenomeGraphs-1.46.0, GenomeInfoDb-1.22.0,
GenomeInfoDbData-1.2.2, GenomicAlignments-1.22.1, GenomicFeatures-1.38.2,
GenomicFiles-1.22.0, GenomicRanges-1.38.0, GenomicScores-1.10.0,
genoset-1.42.0, GEOquery-2.54.1, ggbio-1.34.0, ggcyto-1.14.1, ggplot2-3.3.0,
Glimma-1.14.0, GlobalAncova-4.4.0, globaltest-5.40.0, GO.db-3.10.0,
goseq-1.38.0, GOstats-2.52.0, graph-1.64.0, GSEABase-1.48.0, gsmoothr-0.1.7,
GSVA-1.34.0, Gviz-1.30.3, GWASExactHW-1.01, GWASTools-1.32.0,
HDF5Array-1.14.2, hgu133plus2.db-3.2.3, HMMcopy-1.28.1, Homo.sapiens-1.3.1,
IHW-1.14.0, illuminaio-0.28.0, impute-1.60.0, interactiveDisplayBase-1.24.0,
IRanges-2.20.2, isoband-0.2.1, KEGG.db-3.2.3, KEGGgraph-1.46.0,
KEGGprofile-1.28.0, KEGGREST-1.26.1, LEA-2.8.0, limma-3.42.2,
lpsymphony-1.14.0, lumi-2.38.0, MALDIquant-1.19.3, marray-1.64.0,
maSigPro-1.58.0, MassSpecWavelet-1.52.0, MEDIPS-1.38.0, metagenomeSeq-1.28.2,
metap-1.3, methylumi-2.32.0, Mfuzz-2.46.0, minfi-1.32.0, mixOmics-6.10.8,
MLInterfaces-1.66.2, MotifDb-1.28.0, motifStack-1.30.0, MotIV-1.42.0,
MSnbase-2.12.0, multtest-2.42.0, mutoss-0.1-12, mzID-1.24.0, mzR-2.20.0,
ncdfFlow-2.32.0, NOISeq-2.30.0, oligoClasses-1.48.0, openCyto-1.24.0,
org.Hs.eg.db-3.10.0, OrganismDbi-1.28.0, pcaMethods-1.78.0, perm-1.0-0.0,
PFAM.db-3.10.0, phyloseq-1.30.0, polyester-1.22.0, preprocessCore-1.48.0,
pRoloc-1.26.0, pRolocdata-1.24.0, pRolocGUI-1.20.0, ProtGenerics-1.18.0,
PSCBS-0.65.0, PureCN-1.16.0, quantsmooth-1.52.0, qvalue-2.18.0,
R.devices-2.16.1, R.filesets-2.13.0, R.huge-0.9.0, R.utils-2.9.2,
rappdirs-0.3.1, rARPACK-0.11-0, RBGL-1.62.1, RcppAnnoy-0.0.14, RcppHNSW-0.2.0,
RcppParallel-4.4.4, readr-1.3.1, regioneR-1.18.1, Repitools-1.32.0,
ReportingTools-2.26.0, rGADEM-2.34.1, Rgraphviz-2.30.0, rhdf5-2.30.1,
Rhdf5lib-1.8.0, Rhtslib-1.18.1, Ringo-1.50.0, RNASeqPower-1.26.0,
RProtoBufLib-1.8.0, Rsamtools-2.2.3, Rsolnp-1.16, RSpectra-0.16-0,
Rsubread-2.0.1, rsvd-1.0.3, rtracklayer-1.46.0, S4Vectors-0.24.3, samr-3.0,
SC3-1.14.0, scater-1.14.6, scran-1.14.6, scrime-1.3.5, SeqArray-1.26.2,
seqLogo-1.52.0, SeqVarTools-1.24.0, Seurat-3.1.5, shinyFiles-0.7.5,
ShortRead-1.44.3, sigaR-1.34.0, siggenes-1.60.0, SingleCellExperiment-1.8.0,
sitmo-2.0.1, SNPRelate-1.20.1, SPIA-2.38.0, SSPA-2.26.0,
SummarizedExperiment-1.16.1, sva-3.34.0, sys-3.3, tkWidgets-1.64.0,
truncnorm-1.0-8, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2,
TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0, tximport-1.14.0, uwot-0.1.8,
VariantAnnotation-1.32.0, venn-1.9, vsn-3.54.0, widgetTools-1.64.0,
Wrench-1.4.0, xcms-3.8.1, XVector-0.26.0, zlibbioc-1.32.0
]==])

whatis([==[Description: Bioconductor provides tools for the analysis and coprehension
 of high-throughput genomic data.]==])
whatis([==[Homepage: https://bioconductor.org]==])
whatis([==[URL: https://bioconductor.org]==])
whatis([==[Extensions: admisc-0.5, affy-1.64.0, affycoretools-1.58.4, affyio-1.56.0, AgiMicroRna-2.36.0, ALDEx2-1.18.0, ALL-1.28.0, annaffy-1.58.0, annotate-1.64.0, AnnotationDbi-1.48.0, AnnotationFilter-1.10.0, AnnotationForge-1.28.0, AnnotationHub-2.18.0, aroma.affymetrix-3.2.0, aroma.apd-0.6.0, aroma.core-3.2.1, aroma.light-3.16.0, askpass-1.1, ATACseqQC-1.10.2, ballgown-2.18.0, batchelor-1.2.4, baySeq-2.20.0, beachmat-2.2.1, Biobase-2.46.0, BiocFileCache-1.10.2, BiocGenerics-0.32.0, BiocManager-1.30.10, BiocNeighbors-1.4.1, BiocParallel-1.20.1, BiocSingular-1.2.2, BiocVersion-3.10.1, biomaRt-2.42.0, biomformat-1.14.0, Biostrings-2.54.0, biovizBase-1.34.1, BSgenome-1.54.0, BSgenome.Hsapiens.UCSC.hg19-1.4.0, BSgenome.Hsapiens.UCSC.hg38-1.4.1, BSgenome.Mmusculus.UCSC.mm10-1.4.0, bumphunter-1.28.0, CAMERA-1.42.0, Category-2.52.1, CGHbase-1.46.0, ChIPpeakAnno-3.20.1, clue-0.3-57, ComplexHeatmap-2.2.0, ConsensusClusterPlus-1.50.0, cummeRbund-2.28.0, cytolib-1.8.0, CytoML-1.12.1, dada2-1.14.1, DeconRNASeq-1.28.0, DEGseq-1.40.0, DelayedArray-0.12.2, DelayedMatrixStats-1.8.0, derfinder-1.20.0, derfinderHelper-1.20.0, DESeq2-1.26.0, diffcyt-1.6.6, DNAcopy-1.60.0, dqrng-0.2.1, DropletUtils-1.6.1, dupRadar-1.16.0, DynDoc-1.64.0, EBImage-4.28.1, edgeR-3.28.0, ensembldb-2.10.2, fda-5.1.4, FDb.InfiniumMethylation.hg19-2.2.0, fgsea-1.12.0, flowClust-3.24.0, flowCore-1.52.1, FlowSOM-1.18.0, flowStats-3.44.0, flowViz-1.50.0, flowWorkspace-3.34.1, fs-1.3.1, gcrma-2.58.0, gdsfmt-1.22.0, genefilter-1.68.0, geneLenDataBase-1.22.0, geneplotter-1.64.0, GENESIS-2.16.1, GENIE3-1.8.0, GenomeGraphs-1.46.0, GenomeInfoDb-1.22.0, GenomeInfoDbData-1.2.2, GenomicAlignments-1.22.1, GenomicFeatures-1.38.2, GenomicFiles-1.22.0, GenomicRanges-1.38.0, GenomicScores-1.10.0, genoset-1.42.0, GEOquery-2.54.1, ggbio-1.34.0, ggcyto-1.14.1, ggplot2-3.3.0, Glimma-1.14.0, GlobalAncova-4.4.0, globaltest-5.40.0, GO.db-3.10.0, goseq-1.38.0, GOstats-2.52.0, graph-1.64.0, GSEABase-1.48.0, gsmoothr-0.1.7, GSVA-1.34.0, Gviz-1.30.3, GWASExactHW-1.01, GWASTools-1.32.0, HDF5Array-1.14.2, hgu133plus2.db-3.2.3, HMMcopy-1.28.1, Homo.sapiens-1.3.1, IHW-1.14.0, illuminaio-0.28.0, impute-1.60.0, interactiveDisplayBase-1.24.0, IRanges-2.20.2, isoband-0.2.1, KEGG.db-3.2.3, KEGGgraph-1.46.0, KEGGprofile-1.28.0, KEGGREST-1.26.1, LEA-2.8.0, limma-3.42.2, lpsymphony-1.14.0, lumi-2.38.0, MALDIquant-1.19.3, marray-1.64.0, maSigPro-1.58.0, MassSpecWavelet-1.52.0, MEDIPS-1.38.0, metagenomeSeq-1.28.2, metap-1.3, methylumi-2.32.0, Mfuzz-2.46.0, minfi-1.32.0, mixOmics-6.10.8, MLInterfaces-1.66.2, MotifDb-1.28.0, motifStack-1.30.0, MotIV-1.42.0, MSnbase-2.12.0, multtest-2.42.0, mutoss-0.1-12, mzID-1.24.0, mzR-2.20.0, ncdfFlow-2.32.0, NOISeq-2.30.0, oligoClasses-1.48.0, openCyto-1.24.0, org.Hs.eg.db-3.10.0, OrganismDbi-1.28.0, pcaMethods-1.78.0, perm-1.0-0.0, PFAM.db-3.10.0, phyloseq-1.30.0, polyester-1.22.0, preprocessCore-1.48.0, pRoloc-1.26.0, pRolocdata-1.24.0, pRolocGUI-1.20.0, ProtGenerics-1.18.0, PSCBS-0.65.0, PureCN-1.16.0, quantsmooth-1.52.0, qvalue-2.18.0, R.devices-2.16.1, R.filesets-2.13.0, R.huge-0.9.0, R.utils-2.9.2, rappdirs-0.3.1, rARPACK-0.11-0, RBGL-1.62.1, RcppAnnoy-0.0.14, RcppHNSW-0.2.0, RcppParallel-4.4.4, readr-1.3.1, regioneR-1.18.1, Repitools-1.32.0, ReportingTools-2.26.0, rGADEM-2.34.1, Rgraphviz-2.30.0, rhdf5-2.30.1, Rhdf5lib-1.8.0, Rhtslib-1.18.1, Ringo-1.50.0, RNASeqPower-1.26.0, RProtoBufLib-1.8.0, Rsamtools-2.2.3, Rsolnp-1.16, RSpectra-0.16-0, Rsubread-2.0.1, rsvd-1.0.3, rtracklayer-1.46.0, S4Vectors-0.24.3, samr-3.0, SC3-1.14.0, scater-1.14.6, scran-1.14.6, scrime-1.3.5, SeqArray-1.26.2, seqLogo-1.52.0, SeqVarTools-1.24.0, Seurat-3.1.5, shinyFiles-0.7.5, ShortRead-1.44.3, sigaR-1.34.0, siggenes-1.60.0, SingleCellExperiment-1.8.0, sitmo-2.0.1, SNPRelate-1.20.1, SPIA-2.38.0, SSPA-2.26.0, SummarizedExperiment-1.16.1, sva-3.34.0, sys-3.3, tkWidgets-1.64.0, truncnorm-1.0-8, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0, tximport-1.14.0, uwot-0.1.8, VariantAnnotation-1.32.0, venn-1.9, vsn-3.54.0, widgetTools-1.64.0, Wrench-1.4.0, xcms-3.8.1, XVector-0.26.0, zlibbioc-1.32.0]==])

local root = "/app/software/R-bundle-Bioconductor/3.10-foss-2019b-R-3.6.2"

conflict("R-bundle-Bioconductor")

if not ( isloaded("foss/2019b") ) then
    load("foss/2019b")
end

if not ( isloaded("R/3.6.2-foss-2019b") ) then
    load("R/3.6.2-foss-2019b")
end

if not ( isloaded("HDF5/1.10.5-gompi-2019b") ) then
    load("HDF5/1.10.5-gompi-2019b")
end

if not ( isloaded("MariaDB-connector-c/2.3.7-GCCcore-8.3.0") ) then
    load("MariaDB-connector-c/2.3.7-GCCcore-8.3.0")
end

if not ( isloaded("ncdf4/1.17-foss-2019b") ) then
    load("ncdf4/1.17-foss-2019b")
end

if not ( isloaded("Graphviz/2.42.2-foss-2019b") ) then
    load("Graphviz/2.42.2-foss-2019b")
end

prepend_path("CMAKE_PREFIX_PATH", root)
setenv("EBROOTRMINBUNDLEMINBIOCONDUCTOR", root)
setenv("EBVERSIONRMINBUNDLEMINBIOCONDUCTOR", "3.10")
setenv("EBDEVELRMINBUNDLEMINBIOCONDUCTOR", pathJoin(root, "logs/R-bundle-Bioconductor-3.10-foss-2019b-R-3.6.2-easybuild-devel"))

prepend_path("R_LIBS", root)
-- Built with EasyBuild version 4.2.1
setenv("EBEXTSLISTRMINBUNDLEMINBIOCONDUCTOR", "readr-1.3.1,BiocGenerics-0.32.0,Biobase-2.46.0,S4Vectors-0.24.3,IRanges-2.20.2,GenomeInfoDbData-1.2.2,GenomeInfoDb-1.22.0,AnnotationDbi-1.48.0,zlibbioc-1.32.0,XVector-0.26.0,Biostrings-2.54.0,GenomicRanges-1.38.0,BiocParallel-1.20.1,Rhtslib-1.18.1,Rsamtools-2.2.3,DelayedArray-0.12.2,SummarizedExperiment-1.16.1,GenomicAlignments-1.22.1,ShortRead-1.44.3,graph-1.64.0,affyio-1.56.0,preprocessCore-1.48.0,BiocManager-1.30.10,affy-1.64.0,GO.db-3.10.0,limma-3.42.2,RBGL-1.62.1,org.Hs.eg.db-3.10.0,AnnotationForge-1.28.0,KEGG.db-3.2.3,annaffy-1.58.0,gcrma-2.58.0,oligoClasses-1.48.0,edgeR-3.28.0,PFAM.db-3.10.0,perm-1.0-0.0,baySeq-2.20.0,qvalue-2.18.0,impute-1.60.0,fs-1.3.1,shinyFiles-0.7.5,samr-3.0,DEGseq-1.40.0,hgu133plus2.db-3.2.3,sys-3.3,askpass-1.1,illuminaio-0.28.0,rtracklayer-1.46.0,rappdirs-0.3.1,BiocFileCache-1.10.2,biomaRt-2.42.0,GenomicFeatures-1.38.2,bumphunter-1.28.0,multtest-2.42.0,scrime-1.3.5,siggenes-1.60.0,DynDoc-1.64.0,genoset-1.42.0,NOISeq-2.30.0,Rgraphviz-2.30.0,RNASeqPower-1.26.0,annotate-1.64.0,GSEABase-1.48.0,genefilter-1.68.0,Category-2.52.1,GOstats-2.52.0,BSgenome-1.54.0,VariantAnnotation-1.32.0,interactiveDisplayBase-1.24.0,BiocVersion-3.10.1,AnnotationHub-2.18.0,AnnotationFilter-1.10.0,ProtGenerics-1.18.0,ensembldb-2.10.2,biovizBase-1.34.1,OrganismDbi-1.28.0,ggbio-1.34.0,geneplotter-1.64.0,DESeq2-1.26.0,ReportingTools-2.26.0,Glimma-1.14.0,affycoretools-1.58.4,TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2,Homo.sapiens-1.3.1,BSgenome.Hsapiens.UCSC.hg19-1.4.0,AgiMicroRna-2.36.0,GenomeGraphs-1.46.0,geneLenDataBase-1.22.0,goseq-1.38.0,KEGGREST-1.26.1,KEGGgraph-1.46.0,KEGGprofile-1.28.0,GEOquery-2.54.1,RSpectra-0.16-0,rARPACK-0.11-0,mixOmics-6.10.8,Rhdf5lib-1.8.0,rhdf5-2.30.1,HDF5Array-1.14.2,DelayedMatrixStats-1.8.0,minfi-1.32.0,FDb.InfiniumMethylation.hg19-2.2.0,methylumi-2.32.0,lumi-2.38.0,widgetTools-1.64.0,tkWidgets-1.64.0,Mfuzz-2.46.0,admisc-0.5,venn-1.9,maSigPro-1.58.0,SPIA-2.38.0,Gviz-1.30.3,cummeRbund-2.28.0,GenomicFiles-1.22.0,derfinderHelper-1.20.0,derfinder-1.20.0,polyester-1.22.0,Rsubread-2.0.1,pcaMethods-1.78.0,marray-1.64.0,CGHbase-1.46.0,sigaR-1.34.0,Wrench-1.4.0,lpsymphony-1.14.0,IHW-1.14.0,metagenomeSeq-1.28.2,gdsfmt-1.22.0,SNPRelate-1.20.1,biomformat-1.14.0,phyloseq-1.30.0,ALDEx2-1.18.0,RcppParallel-4.4.4,dada2-1.14.1,LEA-2.8.0,tximport-1.14.0,SingleCellExperiment-1.8.0,beachmat-2.2.1,RcppAnnoy-0.0.14,RcppHNSW-0.2.0,BiocNeighbors-1.4.1,rsvd-1.0.3,BiocSingular-1.2.2,scater-1.14.6,sitmo-2.0.1,dqrng-0.2.1,scran-1.14.6,SC3-1.14.0,clue-0.3-57,ComplexHeatmap-2.2.0,GENIE3-1.8.0,dupRadar-1.16.0,DNAcopy-1.60.0,sva-3.34.0,ballgown-2.18.0,DropletUtils-1.6.1,DeconRNASeq-1.28.0,GSVA-1.34.0,PureCN-1.16.0,globaltest-5.40.0,GlobalAncova-4.4.0,SSPA-2.26.0,vsn-3.54.0,MALDIquant-1.19.3,mzID-1.24.0,mzR-2.20.0,MSnbase-2.12.0,MassSpecWavelet-1.52.0,xcms-3.8.1,CAMERA-1.42.0,fgsea-1.12.0,GWASExactHW-1.01,quantsmooth-1.52.0,GWASTools-1.32.0,SeqArray-1.26.2,SeqVarTools-1.24.0,GENESIS-2.16.1,MLInterfaces-1.66.2,pRoloc-1.26.0,pRolocdata-1.24.0,pRolocGUI-1.20.0,EBImage-4.28.1,GenomicScores-1.10.0,BSgenome.Mmusculus.UCSC.mm10-1.4.0,TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0,regioneR-1.18.1,ChIPpeakAnno-3.20.1,seqLogo-1.52.0,rGADEM-2.34.1,MotIV-1.42.0,MotifDb-1.28.0,motifStack-1.30.0,ATACseqQC-1.10.2,batchelor-1.2.4,gsmoothr-0.1.7,Ringo-1.50.0,R.devices-2.16.1,R.filesets-2.13.0,aroma.light-3.16.0,PSCBS-0.65.0,aroma.core-3.2.1,R.huge-0.9.0,aroma.apd-0.6.0,R.utils-2.9.2,aroma.affymetrix-3.2.0,truncnorm-1.0-8,Rsolnp-1.16,Repitools-1.32.0,BSgenome.Hsapiens.UCSC.hg38-1.4.1,MEDIPS-1.38.0,cytolib-1.8.0,flowCore-1.52.1,mutoss-0.1-12,metap-1.3,uwot-0.1.8,Seurat-3.1.5,ALL-1.28.0,ConsensusClusterPlus-1.50.0,flowViz-1.50.0,ncdfFlow-2.32.0,RProtoBufLib-1.8.0,flowWorkspace-3.34.1,fda-5.1.4,flowStats-3.44.0,flowClust-3.24.0,openCyto-1.24.0,isoband-0.2.1,ggplot2-3.3.0,ggcyto-1.14.1,CytoML-1.12.1,FlowSOM-1.18.0,HMMcopy-1.28.1,diffcyt-1.6.6")
