help([==[

Description
===========
R is a free software environment for statistical computing and graphics.


More information
================
 - Homepage: http://www.r-project.org/


Included extensions
===================
affxparser-1.72.0, affy-1.78.0, affycoretools-1.72.0, affyio-1.70.0,
AgiMicroRna-2.50.0, ALDEx2-1.32.0, ALL-1.42.0, alr3-2.0.8, amap-0.8-19,
annaffy-1.72.0, annotate-1.78.0, AnnotationDbi-1.62.1,
AnnotationFilter-1.24.0, AnnotationForge-1.42.0, AnnotationHub-3.8.0,
apeglm-1.22.1, ArchR-1.0.1, argparser-0.7.1, arsenal-3.6.3, arules-1.7-6,
ASCAT-2.5.2, ash-1.0-15, ashr-2.2-54, aws.ec2metadata-0.2.0, aws.s3-0.3.21,
aws.signature-0.6.0, babelgene-22.9, bamsignals-1.32.0, base64url-1.4,
basilisk-1.12.0, basilisk.utils-1.12.1, batchtools-0.9.17, baySeq-2.31.0,
beachmat-2.16.0, benchmark-0.3-6, bezier-1.1.2, Bhat-0.9-12, biglm-0.9-2.1,
Biobase-2.60.0, BiocFileCache-2.8.0, BiocGenerics-0.46.0, BiocIO-1.10.0,
BiocManager-1.30.21, BiocNeighbors-1.18.0, BiocParallel-1.34.2,
BiocSingular-1.16.0, BiocStyle-2.28.0, BiocVersion-3.17.1, biomaRt-2.56.1,
biomformat-1.28.0, Biostrings-2.68.1, biovizBase-1.48.0, BivarP-1.0,
Bolstad-0.2-41, Bolstad2-1.0-29, bookdown-0.34, BradleyTerry2-1.1-2,
breakaway-4.8.4, BSgenome-1.68.0, BSgenome.Hsapiens.UCSC.hg19-1.4.3,
bumphunter-1.42.0, CATALYST-1.24.0, catdata-1.2.3, Category-2.66.0, ccube-1.0,
CGHbase-1.60.0, ChIPpeakAnno-3.34.1, ChIPseeker-1.36.0, chipseq-1.50.0,
chromVAR-1.22.1, CLONETv2-2.2.1, clusterProfiler-4.8.1, CMA-1.58.0,
CNEr-1.36.0, colorRamps-2.3.1, compare-0.2-6, COMPASS-1.38.1,
ComplexHeatmap-2.16.0, config-0.3.1, ConsensusClusterPlus-1.64.0,
copynumber-1.38.0, cummeRbund-2.42.0, curry-0.1.1, CVXR-1.0-11,
cyclocomp-1.1.0, cytolib-2.12.0, CytoML-2.12.0, DAAG-1.25.4, dada2-1.28.0,
DECIPHER-2.28.0, degreenet-1.3-3, DEGseq-1.54.0, DelayedArray-0.26.3,
DelayedMatrixStats-1.22.1, densvis-1.10.2, depmap-1.14.0, derfinder-1.34.0,
derfinderHelper-1.34.0, descr-1.1.7, DESeq2-1.40.1, dfoptim-2020.10-1,
dials-1.2.0, diamonds-1.0-5, DiceDesign-1.9, DiffBind-3.10.0,
diffusionMap-1.2.0, dir.expiry-1.8.0, DirichletMultinomial-1.42.0,
DNAcopy-1.74.1, DO.db-2.9, doBy-4.6.16, doMPI-0.2.2, DOSE-3.26.1, DOT-0.1,
dqrng-0.3.0, drc-3.0-1, DynDoc-1.78.0, EBImage-4.42.0, ECOSolveR-0.5.5,
edgeR-3.42.4, egor-1.23.3, elasticnet-1.3, enrichplot-1.20.0,
ensembldb-2.24.0, EpiModel-2.3.2, epitools-0.5-10.1, ergm.ego-1.1.0,
etrunct-0.1, europepmc-0.4.1, ExperimentHub-2.8.0, fANCOVA-0.6-1,
faraway-1.0.8, fastseg-1.46.0, fasttime-1.1-0, fda-6.1.4,
FDb.InfiniumMethylation.hg19-2.2.0, fds-1.8, fgsea-1.26.0, filelock-1.0.2,
fit.models-0.64, float-0.3-1, flowClust-3.38.0, flowCore-2.12.0,
FlowSOM-2.8.0, flowStats-4.12.0, flowViz-1.64.0, flowWorkspace-4.12.0,
fontBitstreamVera-0.1.1, fontLiberation-0.1.0, fontquiver-0.2.1,
forestplot-3.1.1, forge-0.2.0, fts-0.9.9.2, gage-2.50.0, gcrma-2.72.0,
gdsfmt-1.36.0, gdtools-0.3.3, genefilter-1.82.1, geneLenDataBase-1.36.0,
geneplotter-1.78.0, GENESIS-2.30.0, GenomeInfoDb-1.36.0,
GenomeInfoDbData-1.2.10, GenomicAlignments-1.36.0, GenomicFeatures-1.52.0,
GenomicFiles-1.36.0, GenomicRanges-1.52.0, GEOquery-2.68.0, gfonts-0.2.0,
ggbio-1.48.0, ggcyto-1.28.0, ggdendro-0.1.23, ggnewscale-0.4.9,
ggpointdensity-0.1.0, ggrastr-1.0.2, ggthemes-4.2.4, ggtree-3.8.0,
ggVennDiagram-1.2.2, Glimma-2.10.0, glmpath-0.98, Gmisc-3.0.2, GO.db-3.17.0,
GOSemSim-2.26.0, goseq-1.52.0, GOstats-2.66.0, GPfit-1.0-8, graph-1.78.0,
GreyListChIP-1.32.0, GSEABase-1.62.0, gson-0.1.0, gsubfn-0.7, GSVA-1.48.1,
Gviz-1.44.0, GWASExactHW-1.01, GWASTools-1.46.0, hapassoc-1.2-9,
haplo.stats-1.9.3, HaploSim-1.8.4.2, HDF5Array-1.28.1, HDO.db-0.99.1,
hdrcde-3.4, hgu133plus2.db-3.13.0, HMMcopy-1.42.0, Homo.sapiens-1.3.1,
hopach-2.60.0, HSMMSingleCell-1.20.0, hypergate-0.8.3, igraphdata-1.0.1,
IHW-1.28.0, IlluminaHumanMethylation450kmanifest-0.4.0, illuminaio-0.42.0,
impute-1.74.1, infer-1.0.4, inferference-1.0.2, InteractionSet-1.28.0,
interactiveDisplayBase-1.38.0, invgamma-1.1, iotools-0.3-2, IRanges-2.34.0,
IRdisplay-1.1, its-1.1.8, karyoploteR-1.26.0, KEGGgraph-1.60.0,
KEGGREST-1.40.0, keyring-1.3.1, kin.cohort-0.7, knockoff-0.3.6,
kyotil-2023.2-2, latentnet-2.10.6, lgr-0.4.4, limma-3.56.2, lintr-3.0.2,
LogicReg-1.6.5, lokern-1.1-10, lpridge-1.0-8, lpsymphony-1.28.1, lumi-2.52.0,
LymphoSeq-1.28.0, LymphoSeqDB-0.99.2, MAGeCKFlute-2.4.0, mapdata-2.3.1,
marray-1.78.0, maSigPro-1.72.0, MAST-1.26.0, MatrixEQTL-2.3,
MatrixExtra-0.1.13, MatrixGenerics-1.12.2, matrixStats-1.0.0, MCMCglmm-2.34,
metagenomeSeq-1.42.0, methylKit-1.26.0, methylumi-2.46.0, Mfuzz-2.60.0,
minfi-1.46.0, mixOmics-6.24.0, mixsqp-0.3-48, mlapi-0.1.1, modeldata-1.1.0,
modelenv-0.1.1, mondate-0.10.02, motifmatchr-1.22.0, MPV-1.62, msa-1.32.0,
msigdbr-7.5.1, multtest-2.56.0, NADA-1.6-1.1, nanotime-0.3.7, ncdfFlow-2.46.0,
ndtv-0.13.3, networksis-2.1-3, NOISeq-2.44.0, numbers-0.8-5, nws-1.7.0.1,
odbc-1.3.4, oligo-1.64.1, oligoClasses-1.62.0, openCyto-2.12.0,
org.Hs.eg.db-3.17.0, org.Mm.eg.db-3.17.0, OrganismDbi-1.42.0, osqp-0.6.0.8,
pack-0.1-1, parsnip-1.1.0, partools-1.1.6, pathview-1.40.0, pbs-1.1,
PBSddesolve-1.13.3, pcaMethods-1.92.0, pdftools-3.3.3, pdist-1.2.1,
pedigree-1.4.2, pedigreemm-0.3-3, perm-1.0-0.2, PFAM.db-3.17.0,
phyloseq-1.44.0, PICS-2.44.0, poisbinom-1.0.1, polyester-1.36.0,
PolynomF-2.0-5, polyreg-0.8.0, poweRlaw-0.70.6, premessa-0.2.6,
preprocessCore-1.62.1, ProtGenerics-1.32.0, psychotools-0.7-3,
qdapRegex-0.7.5, qpdf-1.3.2, qqconf-1.3.2, qtl-1.60, quantoptr-0.1.3,
quantsmooth-1.66.0, QUIC-1.1.1, qvalue-2.32.0, qvcalc-1.0.3, r2d3-0.2.6,
R2jags-0.7-1, rainbow-3.7, rARPACK-0.11-0, rbenchmark-1.0.0, RBGL-1.76.0,
RcppAnnoy-0.0.20, RcppCCTZ-0.2.12, RcppDate-0.0.3, RcppHNSW-0.4.1,
RcppML-0.3.7, RcppNumerical-0.5-0, RcppZiggurat-0.1.6, Rdsdp-1.0.5.2.1,
REDCapR-1.1.0, refGenome-1.7.7, refund-0.1-30, regioneR-1.32.0,
regress-1.3-21, regtools-1.7.0, relations-0.6-13, relevent-1.2-1,
ReportingTools-2.39.0, repr-1.1.6, restfulr-0.0.15, Rfast-2.0.7,
rGADEM-2.48.0, rgenoud-5.9-0.3, Rgraphviz-2.44.0, rhandsontable-0.3.8,
rhdf5-2.44.0, rhdf5filters-1.12.1, Rhdf5lib-1.22.0, Rhpc-0.21-247,
Rhtslib-2.2.0, Ringo-1.64.0, rjags-4-14, rje-1.12.1, Rlabkey-2.11.0,
RLRsim-3.1-8, Rmpi-0.7-1, RNASeqPower-1.40.0, RPostgreSQL-0.7-5,
RPresto-1.4.5, RProtoBufLib-2.12.0, rsample-1.1.1, Rsamtools-2.16.0,
RSclient-0.7-9, rslurm-0.6.2, rsparse-0.5.1, rstanarm-2.21.4, Rsubread-2.14.2,
rsvd-1.0.5, rtracklayer-1.60.0, rvcheck-0.2.1, RVenn-1.1.0, rzmq-0.9.9,
S4Arrays-1.0.4, S4Vectors-0.38.1, samr-3.0, ScaledMatrix-1.8.1, scater-1.28.0,
scattermore-1.2, schoolmath-0.4.1, scidb-3.0.0, scrime-1.3.5, scs-3.2.4,
scuttle-1.10.1, SemiPar-1.0-4.2, SeqArray-1.40.1, seqLogo-1.66.0,
seqminer-8.9, sequenza-3.0.0, SeqVarTools-1.38.0, sets-1.0-24, shinyBS-0.61.1,
shinycssloaders-1.0.0, shinyFiles-0.9.3, ShortRead-1.58.0, siggenes-1.74.0,
SingleCellExperiment-1.22.0, sitmo-2.0.2, slider-0.3.0, SNPassoc-2.1-0,
SNPRelate-1.34.1, snpStats-1.50.0, sodium-1.2.1, sparklyr-1.8.1,
sparseMatrixStats-1.12.0, SPIA-2.52.0, sqldf-0.4-11, squash-1.0.9,
srvyr-1.2.0, ssgraph-1.15, SummarizedExperiment-1.30.2, sva-3.48.0,
systemPipeR-2.6.1, tables-0.9.17, tergmLite-2.6.1, text2vec-0.6.3,
TFBSTools-1.38.0, TFMPvalue-0.0.9, tidymodels-1.1.0, tidyposterior-1.0.0,
tidypredict-0.5, tilingArray-1.78.0, timeSeries-4030.106, TitanCNA-1.38.0,
tkWidgets-1.78.0, treeio-1.24.1, tune-1.1.1,
TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, udunits2-0.13.2.1, uwot-0.1.14,
VariantAnnotation-1.46.0, venn-1.11, vimp-2.3.1, vsn-3.68.0, warp-0.2.0,
wavethresh-4.7.2, webp-1.1.0, widgetTools-1.78.0, workflows-1.1.3,
workflowsets-1.0.1, Wrench-1.18.0, xCell-1.1.0, xmlparsedata-1.0.5,
XVector-0.40.0, yardstick-1.2.0, zCompositions-1.4.0-1, zlibbioc-1.46.0
]==])

whatis([==[Description: R is a free software environment for statistical computing and graphics.]==])
whatis([==[Homepage: http://www.r-project.org/]==])
whatis([==[URL: http://www.r-project.org/]==])
whatis([==[Extensions: affxparser-1.72.0, affy-1.78.0, affycoretools-1.72.0, affyio-1.70.0, AgiMicroRna-2.50.0, ALDEx2-1.32.0, ALL-1.42.0, alr3-2.0.8, amap-0.8-19, annaffy-1.72.0, annotate-1.78.0, AnnotationDbi-1.62.1, AnnotationFilter-1.24.0, AnnotationForge-1.42.0, AnnotationHub-3.8.0, apeglm-1.22.1, ArchR-1.0.1, argparser-0.7.1, arsenal-3.6.3, arules-1.7-6, ASCAT-2.5.2, ash-1.0-15, ashr-2.2-54, aws.ec2metadata-0.2.0, aws.s3-0.3.21, aws.signature-0.6.0, babelgene-22.9, bamsignals-1.32.0, base64url-1.4, basilisk-1.12.0, basilisk.utils-1.12.1, batchtools-0.9.17, baySeq-2.31.0, beachmat-2.16.0, benchmark-0.3-6, bezier-1.1.2, Bhat-0.9-12, biglm-0.9-2.1, Biobase-2.60.0, BiocFileCache-2.8.0, BiocGenerics-0.46.0, BiocIO-1.10.0, BiocManager-1.30.21, BiocNeighbors-1.18.0, BiocParallel-1.34.2, BiocSingular-1.16.0, BiocStyle-2.28.0, BiocVersion-3.17.1, biomaRt-2.56.1, biomformat-1.28.0, Biostrings-2.68.1, biovizBase-1.48.0, BivarP-1.0, Bolstad-0.2-41, Bolstad2-1.0-29, bookdown-0.34, BradleyTerry2-1.1-2, breakaway-4.8.4, BSgenome-1.68.0, BSgenome.Hsapiens.UCSC.hg19-1.4.3, bumphunter-1.42.0, CATALYST-1.24.0, catdata-1.2.3, Category-2.66.0, ccube-1.0, CGHbase-1.60.0, ChIPpeakAnno-3.34.1, ChIPseeker-1.36.0, chipseq-1.50.0, chromVAR-1.22.1, CLONETv2-2.2.1, clusterProfiler-4.8.1, CMA-1.58.0, CNEr-1.36.0, colorRamps-2.3.1, compare-0.2-6, COMPASS-1.38.1, ComplexHeatmap-2.16.0, config-0.3.1, ConsensusClusterPlus-1.64.0, copynumber-1.38.0, cummeRbund-2.42.0, curry-0.1.1, CVXR-1.0-11, cyclocomp-1.1.0, cytolib-2.12.0, CytoML-2.12.0, DAAG-1.25.4, dada2-1.28.0, DECIPHER-2.28.0, degreenet-1.3-3, DEGseq-1.54.0, DelayedArray-0.26.3, DelayedMatrixStats-1.22.1, densvis-1.10.2, depmap-1.14.0, derfinder-1.34.0, derfinderHelper-1.34.0, descr-1.1.7, DESeq2-1.40.1, dfoptim-2020.10-1, dials-1.2.0, diamonds-1.0-5, DiceDesign-1.9, DiffBind-3.10.0, diffusionMap-1.2.0, dir.expiry-1.8.0, DirichletMultinomial-1.42.0, DNAcopy-1.74.1, DO.db-2.9, doBy-4.6.16, doMPI-0.2.2, DOSE-3.26.1, DOT-0.1, dqrng-0.3.0, drc-3.0-1, DynDoc-1.78.0, EBImage-4.42.0, ECOSolveR-0.5.5, edgeR-3.42.4, egor-1.23.3, elasticnet-1.3, enrichplot-1.20.0, ensembldb-2.24.0, EpiModel-2.3.2, epitools-0.5-10.1, ergm.ego-1.1.0, etrunct-0.1, europepmc-0.4.1, ExperimentHub-2.8.0, fANCOVA-0.6-1, faraway-1.0.8, fastseg-1.46.0, fasttime-1.1-0, fda-6.1.4, FDb.InfiniumMethylation.hg19-2.2.0, fds-1.8, fgsea-1.26.0, filelock-1.0.2, fit.models-0.64, float-0.3-1, flowClust-3.38.0, flowCore-2.12.0, FlowSOM-2.8.0, flowStats-4.12.0, flowViz-1.64.0, flowWorkspace-4.12.0, fontBitstreamVera-0.1.1, fontLiberation-0.1.0, fontquiver-0.2.1, forestplot-3.1.1, forge-0.2.0, fts-0.9.9.2, gage-2.50.0, gcrma-2.72.0, gdsfmt-1.36.0, gdtools-0.3.3, genefilter-1.82.1, geneLenDataBase-1.36.0, geneplotter-1.78.0, GENESIS-2.30.0, GenomeInfoDb-1.36.0, GenomeInfoDbData-1.2.10, GenomicAlignments-1.36.0, GenomicFeatures-1.52.0, GenomicFiles-1.36.0, GenomicRanges-1.52.0, GEOquery-2.68.0, gfonts-0.2.0, ggbio-1.48.0, ggcyto-1.28.0, ggdendro-0.1.23, ggnewscale-0.4.9, ggpointdensity-0.1.0, ggrastr-1.0.2, ggthemes-4.2.4, ggtree-3.8.0, ggVennDiagram-1.2.2, Glimma-2.10.0, glmpath-0.98, Gmisc-3.0.2, GO.db-3.17.0, GOSemSim-2.26.0, goseq-1.52.0, GOstats-2.66.0, GPfit-1.0-8, graph-1.78.0, GreyListChIP-1.32.0, GSEABase-1.62.0, gson-0.1.0, gsubfn-0.7, GSVA-1.48.1, Gviz-1.44.0, GWASExactHW-1.01, GWASTools-1.46.0, hapassoc-1.2-9, haplo.stats-1.9.3, HaploSim-1.8.4.2, HDF5Array-1.28.1, HDO.db-0.99.1, hdrcde-3.4, hgu133plus2.db-3.13.0, HMMcopy-1.42.0, Homo.sapiens-1.3.1, hopach-2.60.0, HSMMSingleCell-1.20.0, hypergate-0.8.3, igraphdata-1.0.1, IHW-1.28.0, IlluminaHumanMethylation450kmanifest-0.4.0, illuminaio-0.42.0, impute-1.74.1, infer-1.0.4, inferference-1.0.2, InteractionSet-1.28.0, interactiveDisplayBase-1.38.0, invgamma-1.1, iotools-0.3-2, IRanges-2.34.0, IRdisplay-1.1, its-1.1.8, karyoploteR-1.26.0, KEGGgraph-1.60.0, KEGGREST-1.40.0, keyring-1.3.1, kin.cohort-0.7, knockoff-0.3.6, kyotil-2023.2-2, latentnet-2.10.6, lgr-0.4.4, limma-3.56.2, lintr-3.0.2, LogicReg-1.6.5, lokern-1.1-10, lpridge-1.0-8, lpsymphony-1.28.1, lumi-2.52.0, LymphoSeq-1.28.0, LymphoSeqDB-0.99.2, MAGeCKFlute-2.4.0, mapdata-2.3.1, marray-1.78.0, maSigPro-1.72.0, MAST-1.26.0, MatrixEQTL-2.3, MatrixExtra-0.1.13, MatrixGenerics-1.12.2, matrixStats-1.0.0, MCMCglmm-2.34, metagenomeSeq-1.42.0, methylKit-1.26.0, methylumi-2.46.0, Mfuzz-2.60.0, minfi-1.46.0, mixOmics-6.24.0, mixsqp-0.3-48, mlapi-0.1.1, modeldata-1.1.0, modelenv-0.1.1, mondate-0.10.02, motifmatchr-1.22.0, MPV-1.62, msa-1.32.0, msigdbr-7.5.1, multtest-2.56.0, NADA-1.6-1.1, nanotime-0.3.7, ncdfFlow-2.46.0, ndtv-0.13.3, networksis-2.1-3, NOISeq-2.44.0, numbers-0.8-5, nws-1.7.0.1, odbc-1.3.4, oligo-1.64.1, oligoClasses-1.62.0, openCyto-2.12.0, org.Hs.eg.db-3.17.0, org.Mm.eg.db-3.17.0, OrganismDbi-1.42.0, osqp-0.6.0.8, pack-0.1-1, parsnip-1.1.0, partools-1.1.6, pathview-1.40.0, pbs-1.1, PBSddesolve-1.13.3, pcaMethods-1.92.0, pdftools-3.3.3, pdist-1.2.1, pedigree-1.4.2, pedigreemm-0.3-3, perm-1.0-0.2, PFAM.db-3.17.0, phyloseq-1.44.0, PICS-2.44.0, poisbinom-1.0.1, polyester-1.36.0, PolynomF-2.0-5, polyreg-0.8.0, poweRlaw-0.70.6, premessa-0.2.6, preprocessCore-1.62.1, ProtGenerics-1.32.0, psychotools-0.7-3, qdapRegex-0.7.5, qpdf-1.3.2, qqconf-1.3.2, qtl-1.60, quantoptr-0.1.3, quantsmooth-1.66.0, QUIC-1.1.1, qvalue-2.32.0, qvcalc-1.0.3, r2d3-0.2.6, R2jags-0.7-1, rainbow-3.7, rARPACK-0.11-0, rbenchmark-1.0.0, RBGL-1.76.0, RcppAnnoy-0.0.20, RcppCCTZ-0.2.12, RcppDate-0.0.3, RcppHNSW-0.4.1, RcppML-0.3.7, RcppNumerical-0.5-0, RcppZiggurat-0.1.6, Rdsdp-1.0.5.2.1, REDCapR-1.1.0, refGenome-1.7.7, refund-0.1-30, regioneR-1.32.0, regress-1.3-21, regtools-1.7.0, relations-0.6-13, relevent-1.2-1, ReportingTools-2.39.0, repr-1.1.6, restfulr-0.0.15, Rfast-2.0.7, rGADEM-2.48.0, rgenoud-5.9-0.3, Rgraphviz-2.44.0, rhandsontable-0.3.8, rhdf5-2.44.0, rhdf5filters-1.12.1, Rhdf5lib-1.22.0, Rhpc-0.21-247, Rhtslib-2.2.0, Ringo-1.64.0, rjags-4-14, rje-1.12.1, Rlabkey-2.11.0, RLRsim-3.1-8, Rmpi-0.7-1, RNASeqPower-1.40.0, RPostgreSQL-0.7-5, RPresto-1.4.5, RProtoBufLib-2.12.0, rsample-1.1.1, Rsamtools-2.16.0, RSclient-0.7-9, rslurm-0.6.2, rsparse-0.5.1, rstanarm-2.21.4, Rsubread-2.14.2, rsvd-1.0.5, rtracklayer-1.60.0, rvcheck-0.2.1, RVenn-1.1.0, rzmq-0.9.9, S4Arrays-1.0.4, S4Vectors-0.38.1, samr-3.0, ScaledMatrix-1.8.1, scater-1.28.0, scattermore-1.2, schoolmath-0.4.1, scidb-3.0.0, scrime-1.3.5, scs-3.2.4, scuttle-1.10.1, SemiPar-1.0-4.2, SeqArray-1.40.1, seqLogo-1.66.0, seqminer-8.9, sequenza-3.0.0, SeqVarTools-1.38.0, sets-1.0-24, shinyBS-0.61.1, shinycssloaders-1.0.0, shinyFiles-0.9.3, ShortRead-1.58.0, siggenes-1.74.0, SingleCellExperiment-1.22.0, sitmo-2.0.2, slider-0.3.0, SNPassoc-2.1-0, SNPRelate-1.34.1, snpStats-1.50.0, sodium-1.2.1, sparklyr-1.8.1, sparseMatrixStats-1.12.0, SPIA-2.52.0, sqldf-0.4-11, squash-1.0.9, srvyr-1.2.0, ssgraph-1.15, SummarizedExperiment-1.30.2, sva-3.48.0, systemPipeR-2.6.1, tables-0.9.17, tergmLite-2.6.1, text2vec-0.6.3, TFBSTools-1.38.0, TFMPvalue-0.0.9, tidymodels-1.1.0, tidyposterior-1.0.0, tidypredict-0.5, tilingArray-1.78.0, timeSeries-4030.106, TitanCNA-1.38.0, tkWidgets-1.78.0, treeio-1.24.1, tune-1.1.1, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, udunits2-0.13.2.1, uwot-0.1.14, VariantAnnotation-1.46.0, venn-1.11, vimp-2.3.1, vsn-3.68.0, warp-0.2.0, wavethresh-4.7.2, webp-1.1.0, widgetTools-1.78.0, workflows-1.1.3, workflowsets-1.0.1, Wrench-1.18.0, xCell-1.1.0, xmlparsedata-1.0.5, XVector-0.40.0, yardstick-1.2.0, zCompositions-1.4.0-1, zlibbioc-1.46.0]==])

local root = "/app/software/fhR/4.3.0-foss-2022b"

conflict("fhR")

if not ( isloaded("foss/2022b") ) then
    load("foss/2022b")
end

if not ( isloaded("R/4.3.0-foss-2022b") ) then
    load("R/4.3.0-foss-2022b")
end

if not ( isloaded("libwebp/1.3.0-GCCcore-12.2.0") ) then
    load("libwebp/1.3.0-GCCcore-12.2.0")
end

if not ( isloaded("PostgreSQL/15.2-GCCcore-12.2.0") ) then
    load("PostgreSQL/15.2-GCCcore-12.2.0")
end

if not ( isloaded("JAGS/4.3.2-foss-2022b") ) then
    load("JAGS/4.3.2-foss-2022b")
end

if not ( isloaded("GLPK/5.0-GCCcore-12.2.0") ) then
    load("GLPK/5.0-GCCcore-12.2.0")
end

if not ( isloaded("poppler/23.01.0-GCCcore-12.2.0") ) then
    load("poppler/23.01.0-GCCcore-12.2.0")
end

if not ( isloaded("Graphviz/7.0.6-GCCcore-12.2.0") ) then
    load("Graphviz/7.0.6-GCCcore-12.2.0")
end

if not ( isloaded("NLopt/2.7.1-GCCcore-12.2.0") ) then
    load("NLopt/2.7.1-GCCcore-12.2.0")
end

if not ( isloaded("libGLU/9.0.2-GCCcore-12.2.0") ) then
    load("libGLU/9.0.2-GCCcore-12.2.0")
end

if not ( isloaded("Mesa/22.2.4-GCCcore-12.2.0") ) then
    load("Mesa/22.2.4-GCCcore-12.2.0")
end

if not ( isloaded("cairo/1.17.4-GCCcore-12.2.0") ) then
    load("cairo/1.17.4-GCCcore-12.2.0")
end

if not ( isloaded("ICU/72.1-GCCcore-12.2.0") ) then
    load("ICU/72.1-GCCcore-12.2.0")
end

if not ( isloaded("netCDF/4.9.0-gompi-2022b") ) then
    load("netCDF/4.9.0-gompi-2022b")
end

if not ( isloaded("ZeroMQ/4.3.4-GCCcore-12.2.0") ) then
    load("ZeroMQ/4.3.4-GCCcore-12.2.0")
end

if not ( isloaded("SYMPHONY/5.7.1-foss-2022b") ) then
    load("SYMPHONY/5.7.1-foss-2022b")
end

if not ( isloaded("igraph/0.10.6-foss-2022b") ) then
    load("igraph/0.10.6-foss-2022b")
end

if not ( isloaded("OpenJPEG/2.5.0-GCCcore-12.2.0") ) then
    load("OpenJPEG/2.5.0-GCCcore-12.2.0")
end

if not ( isloaded("UnZip/6.0-GCCcore-12.2.0") ) then
    load("UnZip/6.0-GCCcore-12.2.0")
end

if not ( isloaded("zlib/1.2.12-GCCcore-12.2.0") ) then
    load("zlib/1.2.12-GCCcore-12.2.0")
end

if not ( isloaded("NSS/3.85-GCCcore-12.2.0") ) then
    load("NSS/3.85-GCCcore-12.2.0")
end

if not ( isloaded("unixODBC/2.3.11-foss-2022b") ) then
    load("unixODBC/2.3.11-foss-2022b")
end

prepend_path("CMAKE_PREFIX_PATH", root)
setenv("EBROOTFHR", root)
setenv("EBVERSIONFHR", "4.3.0")
setenv("EBDEVELFHR", pathJoin(root, "logs/fhR-4.3.0-foss-2022b-easybuild-devel"))

prepend_path("R_LIBS_SITE", root)
-- Built with EasyBuild version 4.9.1
setenv("EBEXTSLISTFHR", "perm-1.0-0.2,webp-1.1.0,qpdf-1.3.2,pdftools-3.3.3,scidb-3.0.0,ssgraph-1.15,pdist-1.2.1,rje-1.12.1,float-0.3-1,MatrixExtra-0.1.13,lgr-0.4.4,rsparse-0.5.1,mlapi-0.1.1,text2vec-0.6.3,polyreg-0.8.0,regtools-1.7.0,partools-1.1.6,RPostgreSQL-0.7-5,gsubfn-0.7,sqldf-0.4-11,alr3-2.0.8,sets-1.0-24,psychotools-0.7-3,relations-0.6-13,benchmark-0.3-6,Bhat-0.9-12,biglm-0.9-2.1,Bolstad-0.2-41,Bolstad2-1.0-29,qvcalc-1.0.3,BradleyTerry2-1.1-2,catdata-1.2.3,compare-0.2-6,DAAG-1.25.4,degreenet-1.3-3,Rmpi-0.7-1,doMPI-0.2.2,elasticnet-1.3,tergmLite-2.6.1,srvyr-1.2.0,egor-1.23.3,ergm.ego-1.1.0,EpiModel-2.3.2,ash-1.0-15,hdrcde-3.4,rainbow-3.7,fds-1.8,fda-6.1.4,BiocManager-1.30.21,fit.models-0.64,fts-0.9.9.2,glmpath-0.98,hapassoc-1.2-9,HaploSim-1.8.4.2,arsenal-3.6.3,haplo.stats-1.9.3,igraphdata-1.0.1,repr-1.1.6,IRdisplay-1.1,its-1.1.8,kin.cohort-0.7,latentnet-2.10.6,cyclocomp-1.1.0,xmlparsedata-1.0.5,lintr-3.0.2,LogicReg-1.6.5,lokern-1.1-10,lpridge-1.0-8,mapdata-2.3.1,MCMCglmm-2.34,mondate-0.10.02,MPV-1.62,ndtv-0.13.3,networksis-2.1-3,nws-1.7.0.1,PBSddesolve-1.13.3,pedigree-1.4.2,pedigreemm-0.3-3,PolynomF-2.0-5,qtl-1.60,rjags-4-14,R2jags-0.7-1,regress-1.3-21,relevent-1.2-1,Rhpc-0.21-247,fasttime-1.1-0,Rlabkey-2.11.0,RSclient-0.7-9,rzmq-0.9.9,tables-0.9.17,timeSeries-4030.106,wavethresh-4.7.2,config-0.3.1,forge-0.2.0,r2d3-0.2.6,sparklyr-1.8.1,REDCapR-1.1.0,pack-0.1-1,GWASExactHW-1.01,ASCAT-2.5.2,rslurm-0.6.2,diamonds-1.0-5,shinycssloaders-1.0.0,kyotil-2023.2-2,dfoptim-2020.10-1,BivarP-1.0,argparser-0.7.1,rgenoud-5.9-0.3,inferference-1.0.2,udunits2-0.13.2.1,aws.ec2metadata-0.2.0,aws.signature-0.6.0,aws.s3-0.3.21,shinyBS-0.61.1,descr-1.1.7,ggthemes-4.2.4,zlibbioc-1.46.0,BiocGenerics-0.46.0,graph-1.78.0,Biobase-2.60.0,Rhdf5lib-1.22.0,RProtoBufLib-2.12.0,cytolib-2.12.0,S4Vectors-0.38.1,flowCore-2.12.0,flowViz-1.64.0,IRanges-2.34.0,XVector-0.40.0,GenomeInfoDbData-1.2.10,GenomeInfoDb-1.36.0,Biostrings-2.68.1,KEGGREST-1.40.0,AnnotationDbi-1.62.1,GO.db-3.17.0,gage-2.50.0,GenomicRanges-1.52.0,BiocParallel-1.34.2,Rhtslib-2.2.0,Rsamtools-2.16.0,matrixStats-1.0.0,MatrixGenerics-1.12.2,S4Arrays-1.0.4,DelayedArray-0.26.3,SummarizedExperiment-1.30.2,amap-0.8-19,limma-3.56.2,edgeR-3.42.4,GenomicAlignments-1.36.0,BiocIO-1.10.0,restfulr-0.0.15,rtracklayer-1.60.0,filelock-1.0.2,BiocFileCache-2.8.0,biomaRt-2.56.1,GenomicFeatures-1.52.0,BSgenome-1.68.0,VariantAnnotation-1.46.0,annotate-1.78.0,genefilter-1.82.1,geneplotter-1.78.0,qvalue-2.32.0,impute-1.74.1,shinyFiles-0.9.3,samr-3.0,DEGseq-1.54.0,DESeq2-1.40.1,RBGL-1.76.0,AnnotationForge-1.42.0,Rgraphviz-2.44.0,GSEABase-1.62.0,SingleCellExperiment-1.22.0,sparseMatrixStats-1.12.0,DelayedMatrixStats-1.22.1,rhdf5filters-1.12.1,rhdf5-2.44.0,HDF5Array-1.28.1,beachmat-2.16.0,ScaledMatrix-1.8.1,rsvd-1.0.5,BiocSingular-1.16.0,GSVA-1.48.1,Category-2.66.0,GOstats-2.66.0,ShortRead-1.58.0,base64url-1.4,batchtools-0.9.17,DOT-0.1,systemPipeR-2.6.1,RcppNumerical-0.5-0,apeglm-1.22.1,mixsqp-0.3-48,etrunct-0.1,invgamma-1.1,ashr-2.2-54,GreyListChIP-1.32.0,DiffBind-3.10.0,affyio-1.70.0,preprocessCore-1.62.1,affy-1.78.0,org.Hs.eg.db-3.17.0,org.Mm.eg.db-3.17.0,annaffy-1.72.0,gcrma-2.72.0,affxparser-1.72.0,oligoClasses-1.62.0,oligo-1.64.1,PFAM.db-3.17.0,baySeq-2.31.0,hgu133plus2.db-3.13.0,illuminaio-0.42.0,bumphunter-1.42.0,scrime-1.3.5,multtest-2.56.0,siggenes-1.74.0,DynDoc-1.78.0,NOISeq-2.44.0,RNASeqPower-1.40.0,interactiveDisplayBase-1.38.0,BiocVersion-3.17.1,AnnotationHub-3.8.0,ProtGenerics-1.32.0,AnnotationFilter-1.24.0,ensembldb-2.24.0,biovizBase-1.48.0,OrganismDbi-1.42.0,ggbio-1.48.0,ReportingTools-2.39.0,Glimma-2.10.0,affycoretools-1.72.0,TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2,Homo.sapiens-1.3.1,BSgenome.Hsapiens.UCSC.hg19-1.4.3,AgiMicroRna-2.50.0,geneLenDataBase-1.36.0,goseq-1.52.0,KEGGgraph-1.60.0,GEOquery-2.68.0,msa-1.32.0,rvcheck-0.2.1,treeio-1.24.1,ggtree-3.8.0,LymphoSeqDB-0.99.2,LymphoSeq-1.28.0,rARPACK-0.11-0,mixOmics-6.24.0,minfi-1.46.0,FDb.InfiniumMethylation.hg19-2.2.0,methylumi-2.46.0,lumi-2.52.0,widgetTools-1.78.0,tkWidgets-1.78.0,Mfuzz-2.60.0,venn-1.11,maSigPro-1.72.0,SPIA-2.52.0,Gviz-1.44.0,cummeRbund-2.42.0,GenomicFiles-1.36.0,derfinderHelper-1.34.0,derfinder-1.34.0,polyester-1.36.0,Rsubread-2.14.2,pcaMethods-1.92.0,marray-1.78.0,CGHbase-1.60.0,Wrench-1.18.0,lpsymphony-1.28.1,IHW-1.28.0,metagenomeSeq-1.42.0,gdsfmt-1.36.0,SNPRelate-1.34.1,NADA-1.6-1.1,zCompositions-1.4.0-1,RcppZiggurat-0.1.6,Rfast-2.0.7,ALDEx2-1.32.0,dada2-1.28.0,chipseq-1.50.0,DNAcopy-1.74.1,EBImage-4.42.0,fastseg-1.46.0,hopach-2.60.0,seqLogo-1.66.0,rGADEM-2.48.0,PICS-2.44.0,vsn-3.68.0,Ringo-1.64.0,snpStats-1.50.0,tilingArray-1.78.0,IlluminaHumanMethylation450kmanifest-0.4.0,quantsmooth-1.66.0,GWASTools-1.46.0,SeqArray-1.40.1,SeqVarTools-1.38.0,GENESIS-2.30.0,ncdfFlow-2.46.0,flowWorkspace-4.12.0,TitanCNA-1.38.0,HMMcopy-1.42.0,flowStats-4.12.0,flowClust-3.38.0,openCyto-2.12.0,rbenchmark-1.0.0,pbs-1.1,RLRsim-3.1-8,refund-0.1-30,biomformat-1.28.0,phyloseq-1.44.0,breakaway-4.8.4,rstanarm-2.21.4,modeldata-1.1.0,warp-0.2.0,slider-0.3.0,rsample-1.1.1,DiceDesign-1.9,dials-1.2.0,GPfit-1.0-8,parsnip-1.1.0,modelenv-0.1.1,workflows-1.1.3,yardstick-1.2.0,tune-1.1.1,workflowsets-1.0.1,tidyposterior-1.0.0,tidypredict-0.5,infer-1.0.4,tidymodels-1.1.0,xCell-1.1.0,HSMMSingleCell-1.20.0,regioneR-1.32.0,InteractionSet-1.28.0,ChIPpeakAnno-3.34.1,MatrixEQTL-2.3,curry-0.1.1,doBy-4.6.16,refGenome-1.7.7,faraway-1.0.8,epitools-0.5-10.1,bookdown-0.34,BiocStyle-2.28.0,COMPASS-1.38.1,RcppAnnoy-0.0.20,sitmo-2.0.2,dqrng-0.3.0,hypergate-0.8.3,diffusionMap-1.2.0,ggcyto-1.28.0,CytoML-2.12.0,ALL-1.42.0,ConsensusClusterPlus-1.64.0,colorRamps-2.3.1,ggnewscale-0.4.9,ggpointdensity-0.1.0,scattermore-1.2,FlowSOM-2.8.0,Rdsdp-1.0.5.2.1,knockoff-0.3.6,vimp-2.3.1,numbers-0.8-5,SemiPar-1.0-4.2,squash-1.0.9,iotools-0.3-2,seqminer-8.9,copynumber-1.38.0,sequenza-3.0.0,schoolmath-0.4.1,qdapRegex-0.7.5,CMA-1.58.0,DECIPHER-2.28.0,methylKit-1.26.0,poisbinom-1.0.1,SNPassoc-2.1-0,MAST-1.26.0,odbc-1.3.4,QUIC-1.1.1,RcppDate-0.0.3,RcppCCTZ-0.2.12,nanotime-0.3.7,DO.db-2.9,fgsea-1.26.0,GOSemSim-2.26.0,HDO.db-0.99.1,DOSE-3.26.1,europepmc-0.4.1,enrichplot-1.20.0,gson-0.1.0,clusterProfiler-4.8.1,sva-3.48.0,babelgene-22.9,msigdbr-7.5.1,pathview-1.40.0,ExperimentHub-2.8.0,depmap-1.14.0,MAGeCKFlute-2.4.0,forestplot-3.1.1,Gmisc-3.0.2,bezier-1.1.2,bamsignals-1.32.0,karyoploteR-1.26.0,gfonts-0.2.0,fontBitstreamVera-0.1.1,fontLiberation-0.1.0,fontquiver-0.2.1,gdtools-0.3.3,arules-1.7-6,CLONETv2-2.2.1,RPresto-1.4.5,poweRlaw-0.70.6,CNEr-1.36.0,DirichletMultinomial-1.42.0,TFMPvalue-0.0.9,TFBSTools-1.38.0,ggdendro-0.1.23,chromVAR-1.22.1,RVenn-1.1.0,ggVennDiagram-1.2.2,ChIPseeker-1.36.0,motifmatchr-1.22.0,uwot-0.1.14,ComplexHeatmap-2.16.0,ArchR-1.0.1,quantoptr-0.1.3,fANCOVA-0.6-1,RcppHNSW-0.4.1,BiocNeighbors-1.18.0,drc-3.0-1,scuttle-1.10.1,RcppML-0.3.7,ggrastr-1.0.2,dir.expiry-1.8.0,basilisk.utils-1.12.1,basilisk-1.12.0,densvis-1.10.2,scater-1.28.0,CATALYST-1.24.0,rhandsontable-0.3.8,premessa-0.2.6,sodium-1.2.1,keyring-1.3.1,ECOSolveR-0.5.5,scs-3.2.4,osqp-0.6.0.8,CVXR-1.0-11,qqconf-1.3.2,ccube-1.0")
