help([==[

Description
===========
R is a free software environment for statistical computing and graphics.


More information
================
 - Homepage: http://www.r-project.org/


Included extensions
===================
admisc-0.8, affy-1.64.0, affycoretools-1.58.4, affyio-1.56.0,
AgiMicroRna-2.36.0, ALDEx2-1.18.0, ALL-1.28.0, alr3-2.0.8, amap-0.8-18,
annaffy-1.58.0, annotate-1.64.0, AnnotationDbi-1.48.0,
AnnotationFilter-1.10.0, AnnotationForge-1.28.0, AnnotationHub-2.18.0,
argparser-0.6, ASCAT-2.5.2, aws.ec2metadata-0.2.0, aws.s3-0.3.21,
aws.signature-0.5.2, base64url-1.4, batchtools-0.9.13, bayesplot-1.7.1,
baySeq-2.20.0, benchmark-0.3-6, Bhat-0.9-10, biglm-0.9-1, Biobase-2.46.0,
BiocFileCache-1.10.2, BiocGenerics-0.32.0, BiocManager-1.30.10,
BiocParallel-1.20.1, BiocStyle-2.14.4, BiocVersion-3.10.1, biomaRt-2.42.1,
Biostrings-2.54.0, biovizBase-1.34.1, BivarP-1.0, Bolstad-0.2-40,
Bolstad2-1.0-28, bookdown-0.18, BradleyTerry2-1.1-2, breakaway-3.0,
BSgenome-1.54.0, BSgenome.Hsapiens.UCSC.hg19-1.4.0, bumphunter-1.28.0,
catdata-1.2.2, Category-2.52.1, CGEN-3.22.0, CGHbase-1.46.0,
ChIPpeakAnno-3.20.1, chipseq-1.36.0, colourpicker-1.0, compare-0.2-6,
COMPASS-1.24.1, CompQuadForm-1.4.3, config-0.3, ConsensusClusterPlus-1.50.0,
copula-0.999-20, coxme-2.2-16, cummeRbund-2.28.0, cyclocomp-1.1.0,
cytolib-1.8.0, CytoML-1.12.1, DAAG-1.24, dada2-1.14.1, degreenet-1.3-3,
DEGseq-1.40.0, DelayedArray-0.12.3, DelayedMatrixStats-1.8.0,
derfinder-1.20.0, derfinderHelper-1.20.0, descr-1.1.4, DESeq-1.38.0,
DESeq2-1.26.0, dfoptim-2018.2-1, dials-0.0.6, diamonds-1.0-5,
DiceDesign-1.8-1, DiffBind-2.14.0, diffusionMap-1.2.0, DNAcopy-1.60.0,
doBy-4.6.5, doMPI-0.2.2, dqrng-0.2.1, dygraphs-1.1.1.6, DynDoc-1.64.0,
EBImage-4.28.1, edgeR-3.28.1, elasticnet-1.1.1, ensembldb-2.10.2,
EpiModel-1.7.5, epitools-0.5-10.1, faraway-1.0.7, fastseg-1.32.0,
fasttime-1.0-2, fda-5.1.4, FDb.InfiniumMethylation.hg19-2.2.0,
fit.models-0.5-14, flowClust-3.24.0, flowCore-1.52.1, FlowSOM-1.18.0,
flowStats-3.44.0, flowViz-1.50.0, flowWorkspace-3.34.1, forge-0.2.0,
fts-0.9.9.2, furrr-0.1.0, future.apply-1.5.0, gage-2.36.0, gamm4-0.2-6,
gcrma-2.58.0, gdsfmt-1.22.0, genefilter-1.68.0, geneLenDataBase-1.22.0,
geneplotter-1.64.0, GENESIS-2.16.1, GenomeGraphs-1.46.0, GenomeInfoDb-1.22.1,
GenomeInfoDbData-1.2.2, GenomicAlignments-1.22.1, GenomicFeatures-1.38.2,
GenomicFiles-1.22.0, GenomicRanges-1.38.0, genoset-1.42.0, GEOquery-2.54.1,
ggbio-1.34.0, ggcyto-1.14.1, ggExtra-0.9, ggthemes-4.2.0, ggtree-2.0.4,
Glimma-1.14.0, glmpath-0.98, GO.db-3.10.0, goseq-1.38.0, GOstats-2.52.0,
GPfit-1.0-8, graph-1.64.0, GSEABase-1.48.0, gsl-2.1-6, gsubfn-0.7,
GSVA-1.34.0, Gviz-1.30.3, GWASExactHW-1.01, GWASTools-1.32.0, hapassoc-1.2-8,
haplo.stats-1.7.9, HaploSim-1.8.4, hardhat-0.1.2, HCsnip-1.2.0,
HDF5Array-1.14.4, hgu133plus2.db-3.2.3, HMMcopy-1.28.1, Homo.sapiens-1.3.1,
hopach-2.46.0, HSMMSingleCell-1.6.0, hypergate-0.8.3, IDSpatialStats-0.3.10,
igraphdata-1.0.1, IHW-1.14.0, IlluminaHumanMethylation450kmanifest-0.4.0,
illuminaio-0.28.0, impute-1.60.0, infer-0.5.1, inferference-1.0.0,
interactiveDisplayBase-1.24.0, IRanges-2.20.2, IRdisplay-0.7.0, its-1.1.8,
KEGG.db-3.2.3, KEGGgraph-1.46.0, KEGGprofile-1.28.0, KEGGREST-1.26.1,
kin.cohort-0.7, kinship2-1.8.5, kyotil-2019.11-22, lasso2-1.2-20,
latentnet-2.10.5, leiden-0.3.3, limma-3.42.2, lintr-2.0.1, LogicReg-1.6.2,
lokern-1.1-8, loo-2.2.0, lpridge-1.0-8, lpsymphony-1.14.0, lumi-2.38.0,
LymphoSeq-1.14.0, LymphoSeqDB-0.99.2, mailR-0.4.1, mapdata-2.3.0,
mapproj-1.2.7, marray-1.64.0, maSigPro-1.58.0, MAST-1.12.0, MatrixEQTL-2.3,
MCMCglmm-2.29, metagenomeSeq-1.28.2, methylumi-2.32.0, mets-1.2.7.1,
Mfuzz-2.46.0, minfi-1.32.0, mitools-2.4, mixOmics-6.10.9, mondate-0.10.01.02,
MotIV-1.42.0, MPV-1.55, msa-1.18.0, multtest-2.42.0, ncdf4-1.17,
ncdfFlow-2.32.0, ndtv-0.13.0, networksis-2.1-3, NOISeq-2.30.0, nws-1.7.0.1,
oligoClasses-1.48.0, openCyto-1.24.0, org.Hs.eg.db-3.10.0, OrganismDbi-1.28.0,
pack-0.1-1, packrat-0.5.0, parsnip-0.1.0, partools-1.1.6, patchwork-1.0.0,
pbs-1.1, PBSddesolve-1.12.6, pcaMethods-1.78.0, pdftools-2.2, pdist-1.2,
pedigree-1.4, pedigreemm-0.3-3, perm-1.0-0.0, PFAM.db-3.10.0, PICS-2.30.0,
PKI-0.1-7, polyester-1.22.0, PolynomF-2.0-3, preprocessCore-1.48.0,
ProtGenerics-1.18.0, psychotools-0.5-1, qpdf-1.1, qtl-1.46-2,
quantsmooth-1.52.0, qvalue-2.18.0, qvcalc-1.0.2, r2d3-0.2.3, R2jags-0.5-7,
R2WinBUGS-2.1-21, rappdirs-0.3.1, rARPACK-0.11-0, rbenchmark-1.0.0,
RBGL-1.62.1, RcppAnnoy-0.0.16, RcppParallel-5.0.0, REDCapR-0.11.0,
refGenome-1.7.7, refund-0.1-21, regioneR-1.18.1, regress-1.3-15,
regtools-1.1.0, relations-0.6-9, relevent-1.0-4, ReportingTools-2.26.0,
repr-1.1.0, rGADEM-2.34.1, rgenoud-5.8-3.0, Rglpk-0.6-4, Rgraphviz-2.30.0,
rhdf5-2.30.1, Rhdf5lib-1.8.0, Rhpc-0.19-276, Rhtslib-1.18.1, Ringo-1.50.0,
rjags-4-10, Rlabkey-2.4.0, RLRsim-3.1-6, RNASeqPower-1.26.0,
RPostgreSQL-0.6-2, RProtoBufLib-1.8.0, rsample-0.0.6, Rsamtools-2.2.3,
RSclient-0.7-3, rsconnect-0.8.16, rslurm-0.5.0, RSpectra-0.16-0, rstan-2.19.3,
rstanarm-2.19.3, rstantools-2.0.0, Rsubread-2.0.1, rsvd-1.0.3,
rtracklayer-1.46.0, rvcheck-0.1.8, rzmq-0.9.7, S4Vectors-0.24.4, samr-3.0,
scidb-2.0.0, scrime-1.3.5, sctransform-0.2.1, SeqArray-1.26.2, seqLogo-1.52.0,
SeqVarTools-1.24.1, sets-1.0-18, Seurat-3.1.5, shinyBS-0.61,
shinycssloaders-0.3, shinyFiles-0.8.0, shinyjs-1.1, shinystan-2.5.0,
ShortRead-1.44.3, sigaR-1.34.0, siggenes-1.60.0, SingleCellExperiment-1.10.1,
sitmo-2.0.1, SNPRelate-1.20.1, snpStats-1.36.0, sparklyr-1.2.0, SPIA-2.38.0,
sqldf-0.4-11, ssgraph-1.10, StanHeaders-2.19.2, stringdist-0.9.5.5,
SummarizedExperiment-1.16.1, survey-4.0, systemPipeR-1.20.0, tables-0.8.8,
tidymodels-0.1.0, tidyposterior-0.0.2, tidypredict-0.4.5, tidytree-0.3.3,
tilingArray-1.64.0, timeSeries-3062.100, TitanCNA-1.24.0, tkWidgets-1.64.0,
treeio-1.10.0, tune-0.1.0, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2,
udunits2-0.13, UpSetR-1.4.0, uwot-0.1.8, VariantAnnotation-1.32.0, venn-1.9,
vsn-3.54.0, wavethresh-4.6.8, webp-1.0, widgetTools-1.64.0, workflows-0.1.1,
Wrench-1.4.0, xCell-1.2, xmlparsedata-1.0.3, XVector-0.26.0, yardstick-0.0.6,
zlibbioc-1.32.0
]==])

whatis([==[Description: R is a free software environment for statistical computing and graphics.]==])
whatis([==[Homepage: http://www.r-project.org/]==])
whatis([==[URL: http://www.r-project.org/]==])
whatis([==[Extensions: admisc-0.8, affy-1.64.0, affycoretools-1.58.4, affyio-1.56.0, AgiMicroRna-2.36.0, ALDEx2-1.18.0, ALL-1.28.0, alr3-2.0.8, amap-0.8-18, annaffy-1.58.0, annotate-1.64.0, AnnotationDbi-1.48.0, AnnotationFilter-1.10.0, AnnotationForge-1.28.0, AnnotationHub-2.18.0, argparser-0.6, ASCAT-2.5.2, aws.ec2metadata-0.2.0, aws.s3-0.3.21, aws.signature-0.5.2, base64url-1.4, batchtools-0.9.13, bayesplot-1.7.1, baySeq-2.20.0, benchmark-0.3-6, Bhat-0.9-10, biglm-0.9-1, Biobase-2.46.0, BiocFileCache-1.10.2, BiocGenerics-0.32.0, BiocManager-1.30.10, BiocParallel-1.20.1, BiocStyle-2.14.4, BiocVersion-3.10.1, biomaRt-2.42.1, Biostrings-2.54.0, biovizBase-1.34.1, BivarP-1.0, Bolstad-0.2-40, Bolstad2-1.0-28, bookdown-0.18, BradleyTerry2-1.1-2, breakaway-3.0, BSgenome-1.54.0, BSgenome.Hsapiens.UCSC.hg19-1.4.0, bumphunter-1.28.0, catdata-1.2.2, Category-2.52.1, CGEN-3.22.0, CGHbase-1.46.0, ChIPpeakAnno-3.20.1, chipseq-1.36.0, colourpicker-1.0, compare-0.2-6, COMPASS-1.24.1, CompQuadForm-1.4.3, config-0.3, ConsensusClusterPlus-1.50.0, copula-0.999-20, coxme-2.2-16, cummeRbund-2.28.0, cyclocomp-1.1.0, cytolib-1.8.0, CytoML-1.12.1, DAAG-1.24, dada2-1.14.1, degreenet-1.3-3, DEGseq-1.40.0, DelayedArray-0.12.3, DelayedMatrixStats-1.8.0, derfinder-1.20.0, derfinderHelper-1.20.0, descr-1.1.4, DESeq-1.38.0, DESeq2-1.26.0, dfoptim-2018.2-1, dials-0.0.6, diamonds-1.0-5, DiceDesign-1.8-1, DiffBind-2.14.0, diffusionMap-1.2.0, DNAcopy-1.60.0, doBy-4.6.5, doMPI-0.2.2, dqrng-0.2.1, dygraphs-1.1.1.6, DynDoc-1.64.0, EBImage-4.28.1, edgeR-3.28.1, elasticnet-1.1.1, ensembldb-2.10.2, EpiModel-1.7.5, epitools-0.5-10.1, faraway-1.0.7, fastseg-1.32.0, fasttime-1.0-2, fda-5.1.4, FDb.InfiniumMethylation.hg19-2.2.0, fit.models-0.5-14, flowClust-3.24.0, flowCore-1.52.1, FlowSOM-1.18.0, flowStats-3.44.0, flowViz-1.50.0, flowWorkspace-3.34.1, forge-0.2.0, fts-0.9.9.2, furrr-0.1.0, future.apply-1.5.0, gage-2.36.0, gamm4-0.2-6, gcrma-2.58.0, gdsfmt-1.22.0, genefilter-1.68.0, geneLenDataBase-1.22.0, geneplotter-1.64.0, GENESIS-2.16.1, GenomeGraphs-1.46.0, GenomeInfoDb-1.22.1, GenomeInfoDbData-1.2.2, GenomicAlignments-1.22.1, GenomicFeatures-1.38.2, GenomicFiles-1.22.0, GenomicRanges-1.38.0, genoset-1.42.0, GEOquery-2.54.1, ggbio-1.34.0, ggcyto-1.14.1, ggExtra-0.9, ggthemes-4.2.0, ggtree-2.0.4, Glimma-1.14.0, glmpath-0.98, GO.db-3.10.0, goseq-1.38.0, GOstats-2.52.0, GPfit-1.0-8, graph-1.64.0, GSEABase-1.48.0, gsl-2.1-6, gsubfn-0.7, GSVA-1.34.0, Gviz-1.30.3, GWASExactHW-1.01, GWASTools-1.32.0, hapassoc-1.2-8, haplo.stats-1.7.9, HaploSim-1.8.4, hardhat-0.1.2, HCsnip-1.2.0, HDF5Array-1.14.4, hgu133plus2.db-3.2.3, HMMcopy-1.28.1, Homo.sapiens-1.3.1, hopach-2.46.0, HSMMSingleCell-1.6.0, hypergate-0.8.3, IDSpatialStats-0.3.10, igraphdata-1.0.1, IHW-1.14.0, IlluminaHumanMethylation450kmanifest-0.4.0, illuminaio-0.28.0, impute-1.60.0, infer-0.5.1, inferference-1.0.0, interactiveDisplayBase-1.24.0, IRanges-2.20.2, IRdisplay-0.7.0, its-1.1.8, KEGG.db-3.2.3, KEGGgraph-1.46.0, KEGGprofile-1.28.0, KEGGREST-1.26.1, kin.cohort-0.7, kinship2-1.8.5, kyotil-2019.11-22, lasso2-1.2-20, latentnet-2.10.5, leiden-0.3.3, limma-3.42.2, lintr-2.0.1, LogicReg-1.6.2, lokern-1.1-8, loo-2.2.0, lpridge-1.0-8, lpsymphony-1.14.0, lumi-2.38.0, LymphoSeq-1.14.0, LymphoSeqDB-0.99.2, mailR-0.4.1, mapdata-2.3.0, mapproj-1.2.7, marray-1.64.0, maSigPro-1.58.0, MAST-1.12.0, MatrixEQTL-2.3, MCMCglmm-2.29, metagenomeSeq-1.28.2, methylumi-2.32.0, mets-1.2.7.1, Mfuzz-2.46.0, minfi-1.32.0, mitools-2.4, mixOmics-6.10.9, mondate-0.10.01.02, MotIV-1.42.0, MPV-1.55, msa-1.18.0, multtest-2.42.0, ncdf4-1.17, ncdfFlow-2.32.0, ndtv-0.13.0, networksis-2.1-3, NOISeq-2.30.0, nws-1.7.0.1, oligoClasses-1.48.0, openCyto-1.24.0, org.Hs.eg.db-3.10.0, OrganismDbi-1.28.0, pack-0.1-1, packrat-0.5.0, parsnip-0.1.0, partools-1.1.6, patchwork-1.0.0, pbs-1.1, PBSddesolve-1.12.6, pcaMethods-1.78.0, pdftools-2.2, pdist-1.2, pedigree-1.4, pedigreemm-0.3-3, perm-1.0-0.0, PFAM.db-3.10.0, PICS-2.30.0, PKI-0.1-7, polyester-1.22.0, PolynomF-2.0-3, preprocessCore-1.48.0, ProtGenerics-1.18.0, psychotools-0.5-1, qpdf-1.1, qtl-1.46-2, quantsmooth-1.52.0, qvalue-2.18.0, qvcalc-1.0.2, r2d3-0.2.3, R2jags-0.5-7, R2WinBUGS-2.1-21, rappdirs-0.3.1, rARPACK-0.11-0, rbenchmark-1.0.0, RBGL-1.62.1, RcppAnnoy-0.0.16, RcppParallel-5.0.0, REDCapR-0.11.0, refGenome-1.7.7, refund-0.1-21, regioneR-1.18.1, regress-1.3-15, regtools-1.1.0, relations-0.6-9, relevent-1.0-4, ReportingTools-2.26.0, repr-1.1.0, rGADEM-2.34.1, rgenoud-5.8-3.0, Rglpk-0.6-4, Rgraphviz-2.30.0, rhdf5-2.30.1, Rhdf5lib-1.8.0, Rhpc-0.19-276, Rhtslib-1.18.1, Ringo-1.50.0, rjags-4-10, Rlabkey-2.4.0, RLRsim-3.1-6, RNASeqPower-1.26.0, RPostgreSQL-0.6-2, RProtoBufLib-1.8.0, rsample-0.0.6, Rsamtools-2.2.3, RSclient-0.7-3, rsconnect-0.8.16, rslurm-0.5.0, RSpectra-0.16-0, rstan-2.19.3, rstanarm-2.19.3, rstantools-2.0.0, Rsubread-2.0.1, rsvd-1.0.3, rtracklayer-1.46.0, rvcheck-0.1.8, rzmq-0.9.7, S4Vectors-0.24.4, samr-3.0, scidb-2.0.0, scrime-1.3.5, sctransform-0.2.1, SeqArray-1.26.2, seqLogo-1.52.0, SeqVarTools-1.24.1, sets-1.0-18, Seurat-3.1.5, shinyBS-0.61, shinycssloaders-0.3, shinyFiles-0.8.0, shinyjs-1.1, shinystan-2.5.0, ShortRead-1.44.3, sigaR-1.34.0, siggenes-1.60.0, SingleCellExperiment-1.10.1, sitmo-2.0.1, SNPRelate-1.20.1, snpStats-1.36.0, sparklyr-1.2.0, SPIA-2.38.0, sqldf-0.4-11, ssgraph-1.10, StanHeaders-2.19.2, stringdist-0.9.5.5, SummarizedExperiment-1.16.1, survey-4.0, systemPipeR-1.20.0, tables-0.8.8, tidymodels-0.1.0, tidyposterior-0.0.2, tidypredict-0.4.5, tidytree-0.3.3, tilingArray-1.64.0, timeSeries-3062.100, TitanCNA-1.24.0, tkWidgets-1.64.0, treeio-1.10.0, tune-0.1.0, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, udunits2-0.13, UpSetR-1.4.0, uwot-0.1.8, VariantAnnotation-1.32.0, venn-1.9, vsn-3.54.0, wavethresh-4.6.8, webp-1.0, widgetTools-1.64.0, workflows-0.1.1, Wrench-1.4.0, xCell-1.2, xmlparsedata-1.0.3, XVector-0.26.0, yardstick-0.0.6, zlibbioc-1.32.0]==])

local root = "/app/software/R/3.6.2-foss-2019b-fh1"

conflict("R")

if not ( isloaded("foss/2019b") ) then
    load("foss/2019b")
end

if not ( isloaded("R/3.6.2-foss-2019b") ) then
    load("R/3.6.2-foss-2019b")
end

if not ( isloaded("libwebp/1.0.3-GCCcore-8.3.0") ) then
    load("libwebp/1.0.3-GCCcore-8.3.0")
end

if not ( isloaded("PostgreSQL/10.3-foss-2019b") ) then
    load("PostgreSQL/10.3-foss-2019b")
end

if not ( isloaded("JAGS/4.3.0-foss-2019b") ) then
    load("JAGS/4.3.0-foss-2019b")
end

if not ( isloaded("GLPK/4.65-GCCcore-8.3.0") ) then
    load("GLPK/4.65-GCCcore-8.3.0")
end

if not ( isloaded("poppler/0.87.0-foss-2019b") ) then
    load("poppler/0.87.0-foss-2019b")
end

if not ( isloaded("NSS/3.45-GCCcore-8.3.0") ) then
    load("NSS/3.45-GCCcore-8.3.0")
end

if not ( isloaded("GMime/3.2.7-GCCcore-8.3.0") ) then
    load("GMime/3.2.7-GCCcore-8.3.0")
end

if not ( isloaded("NLopt/2.6.1-GCCcore-8.3.0") ) then
    load("NLopt/2.6.1-GCCcore-8.3.0")
end

if not ( isloaded("libGLU/9.0.1-GCCcore-8.3.0") ) then
    load("libGLU/9.0.1-GCCcore-8.3.0")
end

if not ( isloaded("Mesa/19.1.7-GCCcore-8.3.0") ) then
    load("Mesa/19.1.7-GCCcore-8.3.0")
end

if not ( isloaded("cairo/1.16.0-GCCcore-8.3.0") ) then
    load("cairo/1.16.0-GCCcore-8.3.0")
end

if not ( isloaded("ICU/65.1-GCCcore-8.3.0") ) then
    load("ICU/65.1-GCCcore-8.3.0")
end

if not ( isloaded("netCDF/4.7.1-gompi-2019b") ) then
    load("netCDF/4.7.1-gompi-2019b")
end

if not ( isloaded("ZeroMQ/4.3.2-GCCcore-8.3.0") ) then
    load("ZeroMQ/4.3.2-GCCcore-8.3.0")
end

if not ( isloaded("SYMPHONY/5.6.17-GCCcore-8.3.0") ) then
    load("SYMPHONY/5.6.17-GCCcore-8.3.0")
end

if not ( isloaded("igraph/0.7.1-foss-2019b") ) then
    load("igraph/0.7.1-foss-2019b")
end

if not ( isloaded("OpenJPEG/2.3.1-GCCcore-8.3.0") ) then
    load("OpenJPEG/2.3.1-GCCcore-8.3.0")
end

if not ( isloaded("UnZip/6.0-GCCcore-8.3.0") ) then
    load("UnZip/6.0-GCCcore-8.3.0")
end

if not ( isloaded("zlib/1.2.11-GCCcore-8.3.0") ) then
    load("zlib/1.2.11-GCCcore-8.3.0")
end

prepend_path("CMAKE_PREFIX_PATH", root)
setenv("EBROOTR", root)
setenv("EBVERSIONR", "3.6.2")
setenv("EBDEVELR", pathJoin(root, "logs/R-3.6.2-foss-2019b-fh1-easybuild-devel"))

prepend_path("R_LIBS", root)
-- Built with EasyBuild version 4.2.2
setenv("EBEXTSLISTR", "perm-1.0-0.0,ncdf4-1.17,webp-1.0,RcppParallel-5.0.0,qpdf-1.1,pdftools-2.2,scidb-2.0.0,ssgraph-1.10,pdist-1.2,regtools-1.1.0,partools-1.1.6,RPostgreSQL-0.6-2,gsubfn-0.7,sqldf-0.4-11,alr3-2.0.8,sets-1.0-18,psychotools-0.5-1,relations-0.6-9,benchmark-0.3-6,Bhat-0.9-10,biglm-0.9-1,Bolstad-0.2-40,Bolstad2-1.0-28,qvcalc-1.0.2,BradleyTerry2-1.1-2,catdata-1.2.2,compare-0.2-6,coxme-2.2-16,DAAG-1.24,degreenet-1.3-3,doMPI-0.2.2,elasticnet-1.1.1,EpiModel-1.7.5,fda-5.1.4,fit.models-0.5-14,fts-0.9.9.2,glmpath-0.98,hapassoc-1.2-8,HaploSim-1.8.4,haplo.stats-1.7.9,igraphdata-1.0.1,repr-1.1.0,IRdisplay-0.7.0,its-1.1.8,kin.cohort-0.7,kinship2-1.8.5,lasso2-1.2-20,latentnet-2.10.5,stringdist-0.9.5.5,cyclocomp-1.1.0,xmlparsedata-1.0.3,lintr-2.0.1,LogicReg-1.6.2,lokern-1.1-8,lpridge-1.0-8,mapdata-2.3.0,mapproj-1.2.7,MCMCglmm-2.29,mets-1.2.7.1,mondate-0.10.01.02,MPV-1.55,ndtv-0.13.0,networksis-2.1-3,nws-1.7.0.1,PBSddesolve-1.12.6,pedigree-1.4,pedigreemm-0.3-3,PolynomF-2.0-3,qtl-1.46-2,R2WinBUGS-2.1-21,rjags-4-10,R2jags-0.5-7,regress-1.3-15,relevent-1.0-4,Rglpk-0.6-4,Rhpc-0.19-276,fasttime-1.0-2,Rlabkey-2.4.0,RSclient-0.7-3,rzmq-0.9.7,mitools-2.4,survey-4.0,tables-0.8.8,timeSeries-3062.100,wavethresh-4.6.8,config-0.3,forge-0.2.0,r2d3-0.2.3,rappdirs-0.3.1,sparklyr-1.2.0,REDCapR-0.11.0,pack-0.1-1,GWASExactHW-1.01,StanHeaders-2.19.2,ASCAT-2.5.2,rslurm-0.5.0,CompQuadForm-1.4.3,diamonds-1.0-5,shinycssloaders-0.3,kyotil-2019.11-22,dfoptim-2018.2-1,gsl-2.1-6,copula-0.999-20,BivarP-1.0,argparser-0.6,rgenoud-5.8-3.0,inferference-1.0.0,udunits2-0.13,aws.ec2metadata-0.2.0,aws.signature-0.5.2,aws.s3-0.3.21,shinyBS-0.61,descr-1.1.4,ggthemes-4.2.0,shinyjs-1.1,colourpicker-1.0,ggExtra-0.9,packrat-0.5.0,PKI-0.1-7,rsconnect-0.8.16,zlibbioc-1.32.0,BiocGenerics-0.32.0,graph-1.64.0,BiocManager-1.30.10,cytolib-1.8.0,Biobase-2.46.0,flowCore-1.52.1,flowViz-1.50.0,S4Vectors-0.24.4,IRanges-2.20.2,XVector-0.26.0,Biostrings-2.54.0,KEGGREST-1.26.1,AnnotationDbi-1.48.0,gage-2.36.0,GenomeInfoDbData-1.2.2,GenomeInfoDb-1.22.1,GenomicRanges-1.38.0,BiocParallel-1.20.1,Rhtslib-1.18.1,Rsamtools-2.2.3,DelayedArray-0.12.3,SummarizedExperiment-1.16.1,amap-0.8-18,limma-3.42.2,edgeR-3.28.1,GenomicAlignments-1.22.1,rtracklayer-1.46.0,BiocFileCache-1.10.2,biomaRt-2.42.1,GenomicFeatures-1.38.2,BSgenome-1.54.0,VariantAnnotation-1.32.0,annotate-1.64.0,genefilter-1.68.0,geneplotter-1.64.0,qvalue-2.18.0,impute-1.60.0,shinyFiles-0.8.0,samr-3.0,DESeq-1.38.0,DEGseq-1.40.0,DESeq2-1.26.0,GO.db-3.10.0,RBGL-1.62.1,AnnotationForge-1.28.0,Rgraphviz-2.30.0,GSEABase-1.48.0,Category-2.52.1,GOstats-2.52.0,ShortRead-1.44.3,base64url-1.4,batchtools-0.9.13,systemPipeR-1.20.0,DiffBind-2.14.0,affyio-1.56.0,preprocessCore-1.48.0,affy-1.64.0,org.Hs.eg.db-3.10.0,KEGG.db-3.2.3,annaffy-1.58.0,gcrma-2.58.0,oligoClasses-1.48.0,PFAM.db-3.10.0,baySeq-2.20.0,hgu133plus2.db-3.2.3,illuminaio-0.28.0,bumphunter-1.28.0,scrime-1.3.5,multtest-2.42.0,siggenes-1.60.0,DynDoc-1.64.0,genoset-1.42.0,NOISeq-2.30.0,RNASeqPower-1.26.0,interactiveDisplayBase-1.24.0,BiocVersion-3.10.1,AnnotationHub-2.18.0,ProtGenerics-1.18.0,AnnotationFilter-1.10.0,ensembldb-2.10.2,biovizBase-1.34.1,OrganismDbi-1.28.0,ggbio-1.34.0,ReportingTools-2.26.0,Glimma-1.14.0,affycoretools-1.58.4,TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2,Homo.sapiens-1.3.1,BSgenome.Hsapiens.UCSC.hg19-1.4.0,AgiMicroRna-2.36.0,GenomeGraphs-1.46.0,geneLenDataBase-1.22.0,goseq-1.38.0,KEGGgraph-1.46.0,KEGGprofile-1.28.0,GEOquery-2.54.1,msa-1.18.0,tidytree-0.3.3,rvcheck-0.1.8,treeio-1.10.0,ggtree-2.0.4,LymphoSeqDB-0.99.2,UpSetR-1.4.0,LymphoSeq-1.14.0,RSpectra-0.16-0,rARPACK-0.11-0,mixOmics-6.10.9,Rhdf5lib-1.8.0,rhdf5-2.30.1,HDF5Array-1.14.4,DelayedMatrixStats-1.8.0,minfi-1.32.0,FDb.InfiniumMethylation.hg19-2.2.0,methylumi-2.32.0,lumi-2.38.0,widgetTools-1.64.0,tkWidgets-1.64.0,Mfuzz-2.46.0,admisc-0.8,venn-1.9,maSigPro-1.58.0,SPIA-2.38.0,Gviz-1.30.3,cummeRbund-2.28.0,GenomicFiles-1.22.0,derfinderHelper-1.20.0,derfinder-1.20.0,polyester-1.22.0,Rsubread-2.0.1,pcaMethods-1.78.0,marray-1.64.0,CGHbase-1.46.0,sigaR-1.34.0,HCsnip-1.2.0,Wrench-1.4.0,lpsymphony-1.14.0,IHW-1.14.0,metagenomeSeq-1.28.2,gdsfmt-1.22.0,SNPRelate-1.20.1,ALDEx2-1.18.0,dada2-1.14.1,CGEN-3.22.0,chipseq-1.36.0,DNAcopy-1.60.0,EBImage-4.28.1,fastseg-1.32.0,hopach-2.46.0,seqLogo-1.52.0,rGADEM-2.34.1,MotIV-1.42.0,PICS-2.30.0,vsn-3.54.0,Ringo-1.50.0,snpStats-1.36.0,tilingArray-1.64.0,IlluminaHumanMethylation450kmanifest-0.4.0,quantsmooth-1.52.0,GWASTools-1.32.0,SeqArray-1.26.2,SeqVarTools-1.24.1,GENESIS-2.16.1,ncdfFlow-2.32.0,RProtoBufLib-1.8.0,flowWorkspace-3.34.1,TitanCNA-1.24.0,HMMcopy-1.28.1,flowStats-3.44.0,flowClust-3.24.0,openCyto-1.24.0,rbenchmark-1.0.0,pbs-1.1,gamm4-0.2-6,RLRsim-3.1-6,refund-0.1-21,breakaway-3.0,loo-2.2.0,rstan-2.19.3,rstantools-2.0.0,bayesplot-1.7.1,dygraphs-1.1.1.6,shinystan-2.5.0,rstanarm-2.19.3,furrr-0.1.0,rsample-0.0.6,tidyposterior-0.0.2,yardstick-0.0.6,tidypredict-0.4.5,infer-0.5.1,DiceDesign-1.8-1,dials-0.0.6,parsnip-0.1.0,GPfit-1.0-8,hardhat-0.1.2,workflows-0.1.1,tune-0.1.0,tidymodels-0.1.0,GSVA-1.34.0,xCell-1.2,HSMMSingleCell-1.6.0,regioneR-1.18.1,ChIPpeakAnno-3.20.1,MatrixEQTL-2.3,doBy-4.6.5,refGenome-1.7.7,IDSpatialStats-0.3.10,future.apply-1.5.0,sctransform-0.2.1,leiden-0.3.3,rsvd-1.0.3,faraway-1.0.7,epitools-0.5-10.1,bookdown-0.18,BiocStyle-2.14.4,COMPASS-1.24.1,RcppAnnoy-0.0.16,sitmo-2.0.1,dqrng-0.2.1,uwot-0.1.8,hypergate-0.8.3,diffusionMap-1.2.0,ggcyto-1.14.1,CytoML-1.12.1,ALL-1.28.0,ConsensusClusterPlus-1.50.0,FlowSOM-1.18.0,patchwork-1.0.0,Seurat-3.1.5,SingleCellExperiment-1.10.1,MAST-1.12.0,mailR-0.4.1")
